palantir-java-format/src/test/java/com/palantir/javaformat/java/FileBasedTests.java

```diff
@@ -72,9 +72,7 @@ public FileBasedTests(Class<?> testClass, String testDirName) {
         }
         List<Object[]> testInputs = new ArrayList<>();
         if (!isRecreate()) {
-            assertThat(outputs.size())
-                    .describedAs("unmatched inputs and outputs")
-                    .isEqualTo(inputs.size());
+            assertThat(outputs).describedAs("unmatched inputs and outputs").hasSize(inputs.size());
```

Comment:
heh, needs one more run to reach `hasSameSizeAs`

Reply:
I know I got a little chuckle from that :)